MissionPlanner.sln

```diff
@@ -891,6 +893,14 @@ Global
 		{0B99EBF2-B964-4AAC-9EC5-76353610AAAE}.Release|Any CPU.Build.0 = Release|Any CPU
 		{0B99EBF2-B964-4AAC-9EC5-76353610AAAE}.Release|x86.ActiveCfg = Release|Any CPU
 		{0B99EBF2-B964-4AAC-9EC5-76353610AAAE}.Release|x86.Build.0 = Release|Any CPU
+		{B31E31DB-F30B-4B2E-9FD2-0E80CB10EC83}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{B31E31DB-F30B-4B2E-9FD2-0E80CB10EC83}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{B31E31DB-F30B-4B2E-9FD2-0E80CB10EC83}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{B31E31DB-F30B-4B2E-9FD2-0E80CB10EC83}.Debug|x86.Build.0 = Debug|Any CPU
+		{B31E31DB-F30B-4B2E-9FD2-0E80CB10EC83}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{B31E31DB-F30B-4B2E-9FD2-0E80CB10EC83}.Release|Any CPU.Build.0 = Release|Any CPU
+		{B31E31DB-F30B-4B2E-9FD2-0E80CB10EC83}.Release|x86.ActiveCfg = Release|Any CPU
+		{B31E31DB-F30B-4B2E-9FD2-0E80CB10EC83}.Release|x86.Build.0 = Release|Any CPU

```

Comment:
Question - Are these 3 different from existing build configs?
`0E80CB10EC83}.Release|Any CPU.Build.0 = Release|Any CPU
		{B31E31DB-F30B-4B2E-9FD2-0E80CB10EC83}.Release|x86.ActiveCfg = Release|Any CPU
		{B31E31DB-F30B-4B2E-9FD2-0E80CB10EC83}.Release|x86.Build.0 = Release|Any CPU`

Reply:
These lines get automatically added by visual studio when you add a project to the solution. It's for this stuff, which I don't fully understand, but doesn't really matter.
![image](https://github.com/CarbonixUAV/MissionPlanner_CX/assets/14059190/77a8904c-8a81-44ee-a318-d28991ad890d)